app/assets/stylesheets/page_layouts/feedback-form.css.scss

```diff
@@ -53,8 +61,10 @@
 }
 
 .ui-accordion .ui-accordion-content {
+  background: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.